docs/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Filecoin Virtual Machine"
+description: "The Filecoin Virtual Machine (FVM) allows users to write their own smart-contracts and run them against the Filecoin network. This website contains all the documetation for the FVM project, including examples and reference material to help developers build on the FVM."
+lead: "The Filecoin Virtual Machine (FVM) is an interface that developers can use to deploy smart-contracts on the Filecoin network. FVM contracts will introduce the ability to perfom on-chain computation, or the computation of a state. This is the perfect pairing of blockchain storage and computation that Web3 has been waiting for."
+---
+
+# The Filecoin Virtual Machine
+
+The Filecoin Virtual Machine (FVM) is an interface that enables developers to deploy smart contracts on the Filecoin blockchain. FVM support for smart contracts provides the ability to perform on-chain state computations that guide storage deals from pre-commit through committed states. This is the perfect pairing of blockchain storage and computation that Web3 has been waiting for.
+
+## Why the FVM is useful
+
+There has been a huge demand from the developer community for compatibility with Ethereum and Solidity out of the box. With the FVM, developers can access the massive corpus of audited and battle-tested smart contracts written in Solidity for the Ethereum Virtual Machnine (EVM). The FVM also supports other foreign runtimes and virtual machines, such as JS/SES, LLVm, IR, and BPF.
```

Comment:
> compatibility with Ethereum and Solidity out of the box.

Compatibility with what?

> JS/SES, LLVm, IR, and BPF

Can we get external links to these VMs. Also, that's a lot of acronyms. Any chance we can get the full names listed here?

Reply:
@johnnymatthews I'm researching links for these and SES. I found two links of interest:
- (SES: Securing JavaScript in the real world)[https://medium.com/agoric/ses-securing-javascript-in-the-real-world-4f309e6b66a6] This one seems more useful to our audience. I'm inclined to give this one.
- (npm SES)[https://www.npmjs.com/package/ses] This one is very technical and would be useful for developers.

Reply:
@johnnymatthews Meanwhile, LLVm states something very unfortunate and confusing up front: 
(LLVm Compiler Infrastructure)[https://llvm.org/]. It says, "Despite its name, LLVM has little to do with traditional virtual machines." (Do you want me to engage our engineers  (Raoul, I guess) in conversation as to why we are compatible with it?)

Reply:
For this whole list, the main point of this list is to say that we are not _limited_ to EVM. Instead of giving links to each of these, maybe we could say something like, "Ethereum VM is our main focus, but we will be offering support for other foreign virtual machines." (I'll continue to propose useful links in the meantime.)

Reply:
We discussed this as well in the fvm channel, so see how you like the changes.